cmdLine/src/test/java/topgun/cmdline/TestFileParser.java

```diff
@@ -0,0 +1,121 @@
+package topgun.cmdline;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Paths;
+
+public class TestFileParser {
+
+    String[] param = null;
+    JfrParseCommandLine options = null;
+    CmdLineParser parser = null;
+    Configuration configuration = null;
+
+    @Before
+    public void setup() throws URISyntaxException {
+
+        param = new String[]{
+                "--jfr",
+                getAbsolutePath(),
+                "--outDir",
+                "../",
```

Comment:
not safe - should be a temp directory that is cleaned after

Reply:
actually we are not writing any files so its fine, but should probaby point to the java tmp dir